### DISCUSSION TITLE
Moogle Mod Manager (beta)

### DISCUSSION THREAD

**Discussion by KiameV** [KiameV/moogle-mod-manager#24] [Announcements]
Allows for the management of mods for the Final Fantasy Pixel Remastered games released by Square Enix.

### How To Use
To use, copy the link or download the mod's `mod.xml` or `mod.json` file. Then open up **Moogle Mod Manager**, select the game you'd like to manage mods for, select _Add_ and either from URL or File.

At this point the mod will appear in the list on the left. Selecting the mod will load the preview on the right side. To enable the mod, select the checkbox next to the name of the mod on the left.

### For Modders
`mod def`s (mod.xml/mod.json) can be created within **Moogle Mod Manager**. Once the application is opened, select `Author` from the top then `New Mod`. This will show a _tabbed_ interface for creating the `mod def`. While these fields are being populated, selecting `Validate` will find any issues with the `mod def` and explain anything that's wrong.

#### Example Mod
I will try to get a writeup explaining everything in more detail. An example 'mod' and the `mod def` can be found at:

https://github.com/KiameV/moogle-mod-manager/tree/main/example

Notes:
- If a mod fails to be "Enabled" or "Disabled" properly, the state of the checkbox can become corrupted. To fix, File->Select Game->Select the game again and the checkbox will be fixed. Closing and reopening the app will fix it as well.

<hr /><em>This discussion was created from the release <a href='https://github.com/KiameV/moogle-mod-manager/releases/tag/'>Moogle Mod Manager (beta)</a>.</em>

**Comment by lockecoletreasurehunter** [comment]
Hi! I LOVE this idea. However, I am having issues. With the latest version, when I check the box on any mod, it instantly unchecks it. Any ideas?

**Reply by KiameV** [reply]
hmm interesting. In my development environment it works but when I delete the state (download/installed mods/etc) it breaks as you describe.

I'll get this fixed. Thank you for reporting this!

**Reply by KiameV** [reply]
Fixed with version 0.5.10

**Reply by lockecoletreasurehunter** [reply]
@KiameV as a lifelong FFVI fan, THANK YOU! I'll check it out.
